mmv1/products/apphub/Workload.yaml

```diff
@@ -0,0 +1,241 @@
+    name: displayName
+    description: 'Optional. User-defined name for the Workload. '
+  - !ruby/object:Api::Type::String
+    name: description
+    description: 'Optional. User-defined description of a Workload. '
+  - !ruby/object:Api::Type::NestedObject
+    properties:
+      - !ruby/object:Api::Type::String
+        name: uri
+        description: 'Output only. The underlying compute resource uri. '
+        output: true
+    output: true
+    name: workloadReference
+    description: 'Reference of an underlying compute resource represented by the Workload. '
+  - !ruby/object:Api::Type::NestedObject
+    properties:
+      - !ruby/object:Api::Type::String
+        name: gcpProject
+        description: "Output only. The service project identifier that the underlying
+          cloud resource\nresides in. Empty for non cloud resources. "
+        output: true
+      - !ruby/object:Api::Type::String
+        name: location
+        description: "Output only. The location that the underlying compute resource resides
+          in (e.g\nus-west1). "
+        output: true
+      - !ruby/object:Api::Type::String
+        name: zone
+        description: "Output only. The location that the underlying compute resource resides
+          in if it is zonal\n(e.g us-west1-a). "
+        output: true
+    output: true
+    name: workloadProperties
+    description: 'Properties of an underlying compute resource represented by the Workload. '
+  - !ruby/object:Api::Type::String
+    name: discoveredWorkload
+    diff_suppress_func: 'DiscoveredResourceProjectNumberDiffSuppress'
+    description: 'Required. Immutable. The resource name of the original discovered
+      workload. '
+    required: true
+    immutable: true
+  - !ruby/object:Api::Type::NestedObject
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: type
+            description: 'Criticality type. '
+            required: true
+            values:
+              - :MISSION_CRITICAL
+              - :HIGH
+              - :MEDIUM
+              - :LOW
+        name: criticality
+        description: 'Criticality of the Application, Service, or Workload '
+      - !ruby/object:Api::Type::NestedObject
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: type
+            description: 'Environment type. '
+            required: true
+            values:
+              - :PRODUCTION
+              - :STAGING
+              - :TEST
+              - :DEVELOPMENT
+        name: environment
+        description: 'Environment of the Application, Service, or Workload '
+      - !ruby/object:Api::Type::Array
+        name: developerOwners
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: displayName
+              description: 'Optional. Contact''s name. '
```

Comment:
I'm curious how this renders with the apostophes, but it may need to be cleaned up with a `\`? Same below

Reply:
We've verified this. It renders as expected.